api-reference/20 Data Visualization Widgets/dxChart/1 Configuration/commonAxisSettings/label/position.md

```diff
@@ -6,7 +6,41 @@ default: 'outside'
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies the position of labels relative to the chart or its axis.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+You can specify the following positions for labels on the [valueAxis](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/):
+
+<table class="dx-table">
+    <tr>
+        <th>Value</th>
+        <th>Result</th>
+    </tr>
+    <tr>
+        <td>'inside', 'right'</td>
+        <td><img src="/Content/images/doc/20_1/ChartJS/value-axis-label-right.png" /></td>
```

Comment:
Note that the valid format of image URLs has changed: https://github.com/DevExpress/devextreme-documentation/pull/1351/files

Reply:
Ok, got it. But it doesn't affect the current PR(since all the links are changed now), does it?

Reply:
This is a new link, created after all links were mass-updated, so it should be changed manually.